config/init-https-s2s.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+AVAILABLE_DEPLOYMENTS=" deepcarbon.net
+ deepcarbon.net-staging
+ dcotest.tw.rpi.edu"
+
+case $1 in
+    --d=*|--deployment=*)
+    DEPLOYMENT="${1#*=}"
+
+    case ${DEPLOYMENT} in
+
+        "deepcarbon.net")
+        `curl -L 'https://data.deepcarbon.net:8444/s2s/metadata?type=services&instance=http%3A//data.deepcarbon.net/browsers/ontologies/dco-s2s.ttl%23PersonSearchService'`
```

Comment:
Wrapping in ticks like this will execute the result of the curl command. In this case curl should return {}, so it tries to execute {}.

Perhaps assign to variable and check the result against "{}"

```bash
    case ${DEPLOYMENT} in

        "deepcarbon.net")
        result=`curl -L 'https://data.deepcarbon.net:8444/s2s/metadata?type=services&instance=http%3A//data.deepcarbon.net/browsers/ontologies/dco-s2s.ttl%23PersonSearchService'`
        echo "configured HTTPS s2s for deployment to ${DEPLOYMENT}"
```